### ISSUE TITLE
Problems running multiple HA HomeKit instances

### ISSUE THREAD

**Issue by scottyphillips** [ikalchev/HAP-python#179]
Good morning devs! 

Since my poor RPi 1 model B was getting crushed, I moved most of my Home Assistant to a seperate platform (old core 2 duo running Ubuntu and HASSIO). I kept parts of HA on the RPi for GPIO sensors etc. That means I am now running two HomeKit bridges at the same time. The only issue is that one one of the bridges ever seems to want to work at the same time. I have tried giving them unique names, tried combinations of safe mode, reset them numerous times, tried the fixes mentioned in #176, nothing has seemed to help. Is there anything else that I could try? 

Edit, I've managed to get them both online (not really sure how other then to reboot my phone again). However, not convinced that they are going to remain stable, i'll update this ticket once they fall over. This issue might be related to #176, but my setup was originally working fine after I applied the beta hap_server.py file as per #176

**Comment by scottyphillips** [comment]
it might be related to some sort of bonjour/mDNS issue because for one HomeKit bridge instance only my iPhone is connecting, the Apple TV and my computer seem to refuse to connect.

**Comment by ikalchev** [comment]
I regularly run two instances of HAP in the same network, so it's definitely possible. When you say:

> my iPhone is connecting, the Apple TV and my computer seem to refuse to connect.

Do you mean that your phone pairs with the accessory and after that the TV/comp do not see the device?

**Comment by scottyphillips** [comment]
Yes, so my iPhone managed to connect to both HAP instances for a brief period of time, but then neither my computer, not my ATV were able to connect to either. I will try to recreate the setup today and send you some logs + screenshots. Thanks so much for spending time looking after this module. This HAP module is hands down my favourite part of Home Assistant.

**Comment by ikalchev** [comment]
Are the ATV and mac part of the same apple account? Once someone pairs with an accessory, the other devices of that someone get info about the accessory (keys, etc) through apple. So if your devices are from a different account you won’t be able to see them unless you share them

**Comment by stich86** [comment]
any update? I think to have same issue.. now for me HomeKit and HA is useless :(

**Comment by ikalchev** [comment]
Sorry for the delay. Will look at this as well as soon as I find some time.
